examples/basic-nextjs/package.json

```diff
@@ -22,40 +22,41 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sitecore-cloudsdk/core": "^0.5.1",
-    "@sitecore-cloudsdk/events": "^0.5.1",
-    "@sitecore-content-sdk/nextjs": "^1.0.0",
+    "@sitecore-cloudsdk/core": "^0.5.4",
+    "@sitecore-cloudsdk/events": "^0.5.4",
+    "@sitecore-cloudsdk/personalize": "^0.5.4",
```

Comment:
Do we need to install this package? It's not present in our starter and lives as a peer dependency in content-sdk/nextjs